starknet_py/tests/e2e/account/outside_execution_test.py

```diff
@@ -124,9 +159,49 @@ async def test_account_outside_execution_for_impossible_time_bounds(
         caller=ANY_CALLER,
     )
 
-    tx = await argent_account.execute_v1(calls=[call], max_fee=MAX_FEE)
+    tx = await account.execute_v1(calls=[call], max_fee=MAX_FEE)
 
     with pytest.raises(TransactionRevertedError) as err:
-        await argent_account.client.wait_for_tx(tx.transaction_hash)
+        await argent_account_v040.client.wait_for_tx(tx.transaction_hash)
 
     assert "argent/invalid-timestamp" in err.value.message
+
+
+@pytest.mark.asyncio
+async def test_account_outside_execution_by_itself_is_impossible(
+    argent_account_v040: BaseAccount,
+    map_contract,
+):
+
+    assert any(
+        [
+            await argent_account_v040.supports_interface(
+                OutsideExecutionInterfaceID.V1
+            ),
+            await argent_account_v040.supports_interface(
+                OutsideExecutionInterfaceID.V2
+            ),
+        ]
+    )
```

Comment:
Is this needed?

Reply:
I think yes. But I see your suggest. Removing.